config/main.go

```diff
@@ -49,6 +51,7 @@ var Keys = EntitlementsConfigKeysType{
 	Port:            "PORT",
 	CertsFromEnv:    "CERTS_FROM_ENV",
 	SubsHost:        "SUBS_HOST",
+	ComplianceHost:  "COMPLIANCE_HOST",
```

Comment:
Should we add anything to the README around testing this in dev/qa/stage with certs and updating your `ENT_COMPLIANCE_HOST` locally?

Reply:
for sure, I can add a blurb about this

- [ ] update readme to include instructions around using compliance config